components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/constant/WebSubHubAdapterConstants.java

```diff
@@ -67,27 +80,35 @@ public enum ErrorMessages {
                 "WebSub Hub base URL is not configured."),
         ERROR_PUBLISHING_EVENT_INVALID_PAYLOAD("60002", "Invalid payload provided.",
                 "Event payload cannot be processed."),
-        ERROR_NULL_EVENT_PAYLOAD("60003", "Invalid event payload input ", "Event payload input cannot be null."),
-        ERROR_INVALID_EVENT_URI("60004", "Invalid event URI input", "Event URI input cannot be null or empty."),
-        ERROR_INVALID_EVENT_TOPIC("60005", "Invalid event topic input", "Event topic input cannot be null or empty."),
+        ERROR_NULL_EVENT_PAYLOAD("60003", "Invalid event payload input ",
+                "Event payload input cannot be null."),
+        ERROR_INVALID_EVENT_URI("60004", "Invalid event URI input",
+                "Event URI input cannot be null or empty."),
+        ERROR_INVALID_EVENT_TOPIC("60005", "Invalid event topic input",
+                "Event topic input cannot be null or empty."),
         ERROR_INVALID_EVENT_ORGANIZATION_NAME("60006", "Invalid organization name input",
                 "Event organization name input cannot be null or empty."),
-        ERROR_INVALID_TOPIC("60007", "Invalid WebSub hub topic input", "WebSub hub topic cannot be null " +
-                "or empty."),
-        ERROR_INVALID_WEB_SUB_HUB_BASE_URL("60008", "Invalid WebSub hub base URL input", "WebSub hub base URL cannot " +
-                "be null or empty."),
-        ERROR_INVALID_WEB_SUB_OPERATION("60009", "Invalid WebSub operation input", "WebSub operation cannot be null " +
-                "or empty."),
+        ERROR_INVALID_TOPIC("60007", "Invalid WebSubhub topic input",
```

Comment:
WebSubHub

Check and fix all places

Reply:
fixed with https://github.com/wso2-extensions/identity-event-publishers/pull/13/commits/72d3d953d2029464afac3241adc233d3d96d7897